lib/ui/text/paragraph_builder.cc

```diff
@@ -32,75 +32,75 @@ namespace {
 
 // TextStyle
 
-const int tsLeadingDistributionIndex = 0;
-const int tsColorIndex = 1;
-const int tsTextDecorationIndex = 2;
-const int tsTextDecorationColorIndex = 3;
-const int tsTextDecorationStyleIndex = 4;
-const int tsFontWeightIndex = 5;
-const int tsFontStyleIndex = 6;
-const int tsTextBaselineIndex = 7;
-const int tsTextDecorationThicknessIndex = 8;
-const int tsFontFamilyIndex = 9;
-const int tsFontSizeIndex = 10;
-const int tsLetterSpacingIndex = 11;
-const int tsWordSpacingIndex = 12;
-const int tsHeightIndex = 13;
-const int tsLocaleIndex = 14;
-const int tsBackgroundIndex = 15;
-const int tsForegroundIndex = 16;
-const int tsTextShadowsIndex = 17;
-const int tsFontFeaturesIndex = 18;
-const int tsFontVariationsIndex = 19;
-
-const int tsLeadingDistributionMask = 1 << tsLeadingDistributionIndex;
-const int tsColorMask = 1 << tsColorIndex;
-const int tsTextDecorationMask = 1 << tsTextDecorationIndex;
-const int tsTextDecorationColorMask = 1 << tsTextDecorationColorIndex;
-const int tsTextDecorationStyleMask = 1 << tsTextDecorationStyleIndex;
-const int tsTextDecorationThicknessMask = 1 << tsTextDecorationThicknessIndex;
-const int tsFontWeightMask = 1 << tsFontWeightIndex;
-const int tsFontStyleMask = 1 << tsFontStyleIndex;
-const int tsTextBaselineMask = 1 << tsTextBaselineIndex;
-const int tsFontFamilyMask = 1 << tsFontFamilyIndex;
-const int tsFontSizeMask = 1 << tsFontSizeIndex;
-const int tsLetterSpacingMask = 1 << tsLetterSpacingIndex;
-const int tsWordSpacingMask = 1 << tsWordSpacingIndex;
-const int tsHeightMask = 1 << tsHeightIndex;
-const int tsLocaleMask = 1 << tsLocaleIndex;
-const int tsBackgroundMask = 1 << tsBackgroundIndex;
-const int tsForegroundMask = 1 << tsForegroundIndex;
-const int tsTextShadowsMask = 1 << tsTextShadowsIndex;
-const int tsFontFeaturesMask = 1 << tsFontFeaturesIndex;
-const int tsFontVariationsMask = 1 << tsFontVariationsIndex;
+const int kTsLeadingDistributionIndex = 0;
```

Comment:
Two letter acronyms should be capitalized, so `TS` and `PS`
```suggestion
const int kTsLeadingDistributionIndex = 0;
```
https://dart.dev/guides/language/effective-dart/style#do-capitalize-acronyms-and-abbreviations-longer-than-two-letters-like-words

Reply:
That's the case for Dart but I don't think we've ever enforced it for C++/Obj-C/Java code. I'm not opposed to it though!

Reply:
You're right, I don't really have a preference either.